1.6/alpine/Dockerfile

```diff
@@ -30,11 +31,13 @@ RUN set -ex \
 	&& tar -C /usr/local -xzf golang.tar.gz \
 	&& rm golang.tar.gz \
 	&& cd /usr/local/go/src \
+	&& patch -p2 -i /no-pic.patch \
 	&& ./make.bash \
 	\
-	&& rm -rf /usr/local/bootstrap /usr/local/go/pkg/bootstrap \
+	&& rm -rf /usr/local/bootstrap /usr/local/go/pkg/bootstrap /*.patch \
 	&& apk del .build-deps
 
+
```

Comment:
Can you move this extra blank line up between `ENV` and `COPY` on lines 9 and 10? :pray:  (and please add a comment referencing https://golang.org/issue/14851 while you're at it :+1:)

ie:
```dockerfile
ENV GOLANG_....

# https://golang.org/issue/14851
COPY no-pic.patch /

RUN ...
```